__tests__/intergration.test.js

```diff
@@ -70,12 +70,53 @@ describe("GET /api", () => {
     test("each API property should be correct type", () => {
         return request(app).get("/api").expect(200).then(result => {
             const returnObj = result.body.API;
-            console.log(returnObj);
             for (const key in returnObj) {
                 expect(typeof returnObj[key].description).toBe("string")
                 expect(Array.isArray(returnObj[key].queries)).toBe(true)
                 expect(typeof returnObj[key].exampleResponse).toBe("object")
             }
         })
     })
-});
\ No newline at end of file
+});
+describe("get all articles endpoint", () => {
+    test("Should return a 200 status code", () => {
+        return request(app).get("/api/articles/").expect(200);
+    })
+    test("Should return an array in property 'articles'", () => {
+        return request(app).get("/api/articles/").expect(200).then(result => {
+            expect(result.body).hasOwnProperty("articles");
+            expect(Array.isArray(result.body.articles)).toBe(true);
+        })
+    })
+    test("Should return each article with correct properties", () => {
+        return request(app).get("/api/articles/").expect(200).then(result => {
+            const resultArray = result.body.articles;
+            if (resultArray.length) {
```

Comment:
I don't think this is what Niamh was getting at. If you have 0 results then this test will pass default as the forEach never runs which is incorrect. The API shouldn't be able to pass this test suite by returning an empty array. 

Check that you are actually getting some article objects and haven't got a false positive 👍